superset/config.py

```diff
@@ -115,10 +115,13 @@ def _try_json_readsha(filepath: str, length: int) -> Optional[str]:
 # default viz used in chart explorer
 DEFAULT_VIZ_TYPE = "table"
 
+# maximum row limit that can be set on a query. Will override the row limit defined
+# in all queries (both SQL Lab and chart data queries)
+MAX_GLOBAL_ROW_LIMIT: Optional[int] = None
+# default row limit to apply to queries unless set in the query object
 ROW_LIMIT = 50000
-VIZ_ROW_LIMIT = 10000
-# max rows retreieved when requesting samples from datasource in explore view
-SAMPLES_ROW_LIMIT = 1000
+# default row limit when requesting samples from datasource in explore view
+SAMPLES_ROW_LIMIT = 100
```

Comment:
assuming here that SAMPLES_ROW_LIMIT had no effect except on the frontend

Reply:
thanks for catching, that was a typo!